libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -103,12 +106,40 @@ class MapboxManeuverView : ConstraintLayout {
 
     /**
      * Invoke the method to render primary, secondary, sub instructions and lane information.
-     * @param maneuver Expected
+     * @param maneuvers Expected
+     */
+    fun renderManeuver(maneuvers: Expected<ManeuverError, List<Maneuver>>) {
```

Comment:
Touching on @Guardiola31337's point raised above, we should also offer an overload of:
```kotlin
fun renderManeuver(maneuvers: Expected<ManeuverError, List<Maneuver>>, shieldMap: Map<String, RoadShield?>)
```
so that developers have a chance to avoid the text -> shield icon flash and only display the maneuver with a shield. That's the only way for them to actually have a choice, otherwise, they'd always have to display the text first.